client/app/components/tagging/tagging.directive.js

```diff
@@ -144,8 +144,8 @@
         return {id: tag.id,
                 category: {id: tag.category.id},
                 categorization: {
-                  display_name: tag.categorization.display_name,
-                }
+                  displayName: tag.categorization.display_name,
+                },
```

Comment:
Hi @AllenBW , how do you want to handle this?  I can post the code change here, or can I somehow do a commit on my clone of this branch and update your PR?!  (I haven't done something like that before)